libs/sx1302_hal/Makefile

```diff
@@ -0,0 +1,120 @@
+#
+# Copyright (C) 2022 TDT AG <development@tdt.de>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sx1302_hal
+PKG_VERSION:=2.1.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/Lora-net/sx1302_hal.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=4b42025d1751e04632c0b04160e0d29dbbb222a5
+PKG_SOURCE_DATE:=2021-04-22
+PKG_MIRROR_HASH:=8733a2cf594555c17529c682f79612d178a7f473afdaf962c7bfd65555363b63
```

Comment:
This does not look good to me. In the first place, you specified the exact version, but then you used ``PKG_SOURCE_DATE``, ``PKG_SOURCE_VERSION``.

No need to check out sources from Git, if they are using tags and releases. Refer to other Makefiles, which they are using e.g. codeload